--- dmsquash-live-root.sh	2022-04-15 11:10:23.871738446 +0200
+++ "dmsquash-live-root-patched.sh"	2022-04-15 11:10:18.161665251 +0200
@@ -294,6 +294,7 @@
     losetup -r "$SQUASHED_LOOPDEV" $SQUASHED
     mkdir -m 0755 -p /run/initramfs/squashfs
     mount -n -t squashfs -o ro "$SQUASHED_LOOPDEV" /run/initramfs/squashfs
+    mkdir -p /run/initramfs/squashfs/proc
 
     if [ -d /run/initramfs/squashfs/LiveOS ]; then
         if [ -f /run/initramfs/squashfs/LiveOS/rootfs.img ]; then
